pages/portfolio.html

```diff
@@ -4,9 +4,76 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Amber's Portfolio</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
+  <link href="../styles/portfolio.css" rel="stylesheet"/>
+  <link href='https://fonts.googleapis.com/css?family=Amatic SC' rel='stylesheet'>
 </head>
+
+
 <body>
-  
-</body>
-</html>
\ No newline at end of file
+    <section>
+        <nav class="navigation">
+            <a href="index.html" >Home</a>
+            <a href="about.html" >About</a>
+            <a href="portfolio.html" >Portfolio</a>
+        </nav>
+    </section>
+
+
+    <div class="container">
+      <div class="wrapper">
+        <div class="item" style="background:rgb(200, 245, 244)">
```

Comment:
Style: You should avoid using the `style` tag.  

It overrides stylesheets and can cause bugs in larger projects.